ruby/ext/helix_runtime/native/helix_runtime.h

```diff
@@ -21,6 +21,8 @@ int HELIX_TYPE(VALUE v);
 VALUE HELIX_INT2FIX(int c_int);
 VALUE HELIX_FIX2INT(VALUE fix);
 
+VALUE helix_rb_utf8_str_new(const char* str, long len);
```

Comment:
@chancancode I went with this name because it seemed to follow the existing naming convention of HELIX_+<MRI macro name>, but in lowercase because the MRI macro is also spelled in lowercase.